cloud-storage-uploader/README.md

```diff
@@ -0,0 +1,194 @@
+<!--
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# cloud-storage-uploader
+
+This action uploads files/folders to a [GCS][gcs] bucket. This is useful when
+you want upload build artifacts from your workflow.
+
+Paths to files that are successfully uploaded are set as output variables and can be
+used in subsequent steps.
+
+## Prerequisites
+
+- This action requires Google Cloud credentials that are authorized to upload
+  blobs to the specified bucket. See the Authorization section below for more
+  information.
+
+## Usage
+
+### For uploading a file
+
+```yaml
+steps:
+  - id: upload-file
+    uses: GoogleCloudPlatform/github-actions/cloud-storage-uploader@master
+    with:
+      path: /path/to/file
+      destination: bucket-name/file
+
+  # Example of using the output
+  - id: uploaded-files
+    uses: foo/bar@master
+    env:
+      file: ${{steps.upload-file.outputs.uploaded}}
+```
+
+The file will be uploaded to `gs://bucket-name/file`
+
+### For uploading a folder
+
+```yaml
+steps:
+  - id: upload-files
+    uses: GoogleCloudPlatform/github-actions/cloud-storage-uploader@master
+    with:
+      path: /path/to/folder
+      destination: bucket-name
+
+  # Example of using the output
+  - id: uploaded-files
+    uses: foo/bar@master
+    env:
+      files: ${{steps.upload-files.outputs.uploaded}}
+```
+
+If the folder has the following structure:
+
+```
+.
+└── folder
+    ├── file1
+    └── folder2
+        └── file2
+```
+
+The files will be uploaded to `gs://bucket-name/folder/file1`,`gs://bucket-name/folder/folder2/file2`
+
+## Inputs
+
+- `path`: (Required) The path to a file or folder that should be uploaded
```

Comment:
Add "inside the build step" or "inside the action's filesystem"

Reply:
Done